app/src/main/java/org/systers/mentorship/utils/SingleLiveEvent.kt

```diff
@@ -0,0 +1,59 @@
+package org.systers.mentorship.utils
```

Comment:
I've used it to notify the user with a snackbar when the API call for home statistics fails. It's best to use SingleLiveEvent instead of LiveData for this, because it makes sure that Snackbar/Toast does not reappear on the screen in the event of a configuration change.

I got the idea here:
[LiveData with SnackBar, Navigation and other events (the SingleLiveEvent case)](https://medium.com/androiddevelopers/livedata-with-snackbar-navigation-and-other-events-the-singleliveevent-case-ac2622673150)